crates/fluvio-stream-model/Cargo.toml

```diff
@@ -27,9 +27,15 @@ once_cell = "1.5"
 k8-types = { version = "0.6.0", optional = true }
 
 [dev-dependencies]
+
 tokio = { version = "1.3.0", features = ["macros"] }
 fluvio-future = { version = "0.4.0", features = ["fixture"] }
 fluvio-stream-model = { path = ".", features = ["fixture"] }
+async-std = { version = "1.8.0", default-features = false, features = [
```

Comment:
Do we still ned this?

Reply:
I think so, as one of the tests is using the `async_std::task::JoinHandle` returned by `fluvio_future::spawn`.


 `fluvio_future::spawn` is a reexport of `async_std..::spawn` or `...::spawn_local` depending on arch. `fluvio_future` doesn't also export the `JoinHandle` however, hence the need to import directly.

Reply:
got it.